charts/tfy-llm-gateway/values.yaml

```diff
@@ -46,6 +46,8 @@ global:
     registry: "tfy.jfrog.io"
   ## @param global.namespaceOverride Namespace override (defaults to .Release.Namespace if not set)
   namespaceOverride: ""
+  ## @param global.redisEnabled Enable Redis dependency globally
+  redisEnabled: false
```

Comment:
move this to under the redis section

```
redis:
  enabled: false
```

the global section is used to pass in values from a parent chart